dev/com.ibm.websphere.appserver.features/visibility/public/grpc-1.0/io.openliberty.grpc-1.0.feature

```diff
@@ -12,6 +12,7 @@ Subsystem-Version: 1.0.0
 Subsystem-Name: gRPC 1.0
 -bundles=\
   io.openliberty.grpc.1.0.internal,\
+  io.openliberty.grpc.1.0.internal.monitor,\
```

Comment:
I'm not sure you want to pull these two bundles into both this feature and the auto feature above?? Makes the auto feature a little redundant?